t/shdict.t

```diff
@@ -999,3 +999,179 @@ number expected, got string
 --- no_error_log
 [alert]
 [crit]
+
+
+
+=== TEST 31: decr int
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua '
```

Comment:
@p0pr0ck5 Added the above changes in https://github.com/p0pr0ck5/lua-resty-core/pull/1